driver.go

```diff
@@ -547,9 +549,9 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 	}
 
 	if driverConfig.MemorySwap != "" {
-		swap, err := memoryInBytes(driverConfig.MemorySwap)
-		if err != nil {
-			return nil, nil, err
+		swap, memErr := memoryInBytes(driverConfig.MemorySwap)
+		if memErr != nil {
+			return nil, nil, memErr
```

Comment:
Is using `err` considered variable shadowing in cases like this?

Reply:
Yup; I'm not sure being this strict about shadowing will turn out for the best, but figured I'd give a try while doing some work on this repo